### ISSUE TITLE
Speed Test Problem!

### ISSUE THREAD

**Issue by Joejwu** [Megvii-BaseDetection/YOLOX#596]
I use the command:
python tools/eval.py -n  yolox-l -c weights/yolox_l.pth -b 1 -d 1 --conf 0.001 --fp16 --fuse

here's the result:

2021-08-26 21:49:07 | INFO     | yolox.data.datasets.coco:45 - loading annotations into memory...
2021-08-26 21:49:08 | INFO     | yolox.data.datasets.coco:45 - Done (t=0.58s)
2021-08-26 21:49:08 | INFO     | pycocotools.coco:89 - creating index...
2021-08-26 21:49:08 | INFO     | pycocotools.coco:89 - index created!
2021-08-26 21:49:20 | INFO     | __main__:156 - loading checkpoint from weights/yolox_l.pth
2021-08-26 21:49:23 | INFO     | __main__:160 - loaded checkpoint done.
2021-08-26 21:49:23 | INFO     | __main__:166 - 	Fusing model...
/home/wuqiaojun/anaconda3/envs/mytorch/lib/python3.8/site-packages/torch/nn/modules/module.py:561: UserWarning: The .grad attribute of a Tensor that is not a leaf Tensor is being accessed. Its .grad attribute won't be populated during autograd.backward(). If you indeed want the gradient for a non-leaf Tensor, use .retain_grad() on the non-leaf Tensor. If you access the non-leaf Tensor by mistake, make sure you access the leaf Tensor instead. See github.com/pytorch/pytorch/pull/30531 for more information.
  if param.grad is not None:
100%|##########| 5000/5000 [03:55<00:00, 21.27it/s]
2021-08-26 21:53:19 | INFO     | yolox.evaluators.coco_evaluator:171 - Evaluate in main process...
2021-08-26 21:53:22 | INFO     | yolox.evaluators.coco_evaluator:204 - Loading and preparing results...
2021-08-26 21:53:24 | INFO     | yolox.evaluators.coco_evaluator:204 - DONE (t=2.01s)
2021-08-26 21:53:24 | INFO     | pycocotools.coco:365 - creating index...
2021-08-26 21:53:25 | INFO     | pycocotools.coco:365 - index created!
Running per image evaluation...
Evaluate annotation type *bbox*
COCOeval_opt.evaluate() finished in 10.48 seconds.
Accumulating evaluation results...
COCOeval_opt.accumulate() finished in 1.87 seconds.
2021-08-26 21:53:39 | INFO     | __main__:187 - 
**Average forward time: 26.76 ms, Average NMS time: 15.83 ms, Average inference time: 42.59 ms**
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.498
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.680
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.539
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.322
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.550
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.651
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.378
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.610
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.649
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.467
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.705
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.798

Why does it take much longer than the paper says?
I ran the speed test with the weights I trained myself, and the results were similar!
